app/web/markdown/blog/2024/09/08/summer-2024-updates.md

```diff
@@ -0,0 +1,98 @@
+* **Strong Verification**: we are working on a new identity verification feature called "Strong Verification" that allows you to cryptographically verify your passport information. It's currently available as a preview feature and we'll be building out the rest of the frontend implementation soon.
+* **Profile badges**: we've added profile badges for donors, verified users (passport or phone), volunteers (past and present), and so on. We'll add a few more as we go, but these are a nice little addition to recognize our volunteers and supporters. If you want a "past volunteer" badge, please see the section at the end of this post!
+* **Travel map on profile** shows the places you've visited and lived according to what you fill out, and adds some more color to the profiles!
+
+There's also a few less visible or less exciting ones:
+
+* **We removed all password optinality in the backend**: there were some relics of the old password-optional regime that have been removed, and we also simplified the login form to show the username and password inputs straight away.
+* **A variety of behind-the-scenes upgrades**: we've also done a lot of work that's not very visible. There was a really gnarly bug that was causing havok for a week or so but that we managed to squash last Sunday. We've also created a bunch of monitoring and observability infrastructure that has helped us speed up some parts of the platform that, you might notice it's a bit more snappy now (but there's more to come)! We made some upgrades to the infrastructure that helps us run the project. Finally we also built some new admin tooling that helps mods keep the platform fun and friendly with much less effort!
+* **New support infrastrucutre**: we switched out our support platform from a very old and flaky open source platform to a much newer and better platform called Chatwoot! We also switched from using a commercial mailing list provider to a self-hosted platform that will save us a bunch of money.
+* **You now "decline" requests** instead of "rejecting" them, because this sounds friendlier and Couchers is a friendly place :).
+
+
+## Celebrating our community of 43k Couchers!
+
+Couchers is continuing to grow quickly with over 43,000 registered Couchers! Our most important goal right now is to get more couch surfers into our community and onto the platform.
+
+You can help us with this goal by sharing the platform and spreading the word, as well as continuing to use the platform, provide feedback, and helping us grow!
+
+
+## Our long-term strategic goal: the safest couch surfing platform
+
+For a long time our marketing has centered around being "like CouchSurfing™, but better". While this is still the core vision and has resonated strongly with "CouchSurfing™ refugees", we believe we now need to do some rethinking of our branding. We are now approaching a point where we need to appeal to fist time couch surfers and better explain what sets us apart from other platforms.
```

Comment:
```suggestion
For a long time our marketing has centered around being "like CouchSurfing™, but better". While this is still the core vision and has resonated strongly with current and former CouchSurfing™ users, we believe we need to explain what sets us apart from CouchSurfing™ and other platforms, and also better appeal to fist-time couch surfers.
```

Reply:
I like the "refugees" phrase, but updated anyway :)